.gitlab-ci.yml

```diff
@@ -191,11 +54,14 @@ Coverage:
     max: 2
     when: script_failure
 
+
+# TODO: Migrate Docker Publish tasks to GitHub
+# See https://github.com/rigetti/pyquil/issues/1548
```

Comment:
Issue & comment seem limited to docker - should this PyPI task itself be removed?

Reply:
Yeah, good point, I only left it in because the docker tasks were dependent on it, but better to remove the dependency and disable the docker stuff instead.